invenio_cli/helpers/commands.py

```diff
@@ -151,8 +151,25 @@ def _ensure_containers_running(self):
         docker_helper.start_containers()
         # TODO: Find faster way to procede when containers are ready
         time.sleep(30)  # Give time to the containers to start properly
+    
+    def destroy(self):
+        """Destroys env, containers."""
+        command = [
+            'pipenv', '--rm'
+            ]
+        subprocess.run(command, check=True)
+        click.secho('Destroyed Venv...', fg='green')
+
+        docker_helper = DockerHelper(
+            self.cli_config.get_project_shortname(),
+            local=True)
+        """Stop containers"""
+        docker_helper.stop_containers()
+        """Destroy containers"""
+        docker_helper.destroy_containers()
+        click.secho('Destroyed containers...', fg='green')
 
-    def services(self, force):
+    def services(self, force, remove):
```

Comment:
Why is the argument needed? It is not used anywhere in the function + there is no option (click input) of it.

Reply:
It's not needed, It was on a different implementation and I forgot to remove it. Nice catch!